pymc3/distributions/multivariate.py

```diff
@@ -775,9 +796,10 @@ class Wishart(Continuous):
     This distribution is unusable in a PyMC3 model. You should instead
     use LKJCholeskyCov or LKJCorr.
     """
+    rv_op = wishart
 
-    def __init__(self, nu, V, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    @classmethod
+    def dist(cls, nu, V, *args, **kwargs):
         warnings.warn(
```

Comment:
Looks like @twiecki fixed the distribution in https://github.com/pymc-devs/pymc3/pull/701 ?
Does that mean we can remove the warning, or at least "downgrade" the warning into a recommendation to use LKJ dists instead?

**Disclaimer:** I have no clue what either distributions are 😅

Reply:
I think #701 adds the `WishartBartlett` as a solution for that, I updated the warning to recommend use of  `WishartBartlett` instead of `LKJCorr` or `LKJCholeskyCov`.

cc @twiecki Is this correct ?

Reply:
No, LKJCorr should be used. `Wishart` itself is basically unusable, so if a user really wants that distribution, they can only use `WishartBartlett`, but they should always use either `LKJCorr` or `LKJCholeskyCov`.

Reply:
Let's revert the warning? Other than this PR looks pretty ready

Reply:
I don't think it would appear twice. And implementing a `__new__` just because of this warning seems a little too much.
Let's just rephrase to mention the entire recommendation from above and then go for it.

Reply:
Or like this. (I changed as I was writing the above.)